src/state/reducerActions.ts

```diff
@@ -0,0 +1,32 @@
+import { Movie } from '../Types/MovieTypes';
+
+enum reducerActions {
```

Comment:
It should be changed according to the TS style guide. 

enum MovieListAction {
  HandleMovies = 'NAME_OF_YOUR_ACTION'
}

More info:

### [Enums](https://www.typescriptlang.org/docs/handbook/enums.html)

#### [Microsoft notes](https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN)

- Use Pascal case for Enum types and keys.
- Use abbreviations sparingly.
- Do not use a `Enum` suffix on Enum type names.
- Use a **singular** name for **most Enum types**, but use a **plural** name for Enum types that are **bit fields**.
- Always add the FlagsAttribute to a bit field Enum type.

Reply:
Changed naming of this enum.